net-nntp/sabnzbd/sabnzbd-3.6.1.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..11} )
+PYTHON_REQ_USE="sqlite"
+
+inherit python-single-r1 systemd
+
+MY_PV="${PV/_rc/RC}"
+MY_PV="${MY_PV//_pre*}"
+
+MY_P="${PN/sab/SAB}-${MY_PV}"
+
+DESCRIPTION="Binary newsgrabber with web-interface"
+HOMEPAGE="https://sabnzbd.org/"
+SRC_URI="https://github.com/sabnzbd/sabnzbd/releases/download/${MY_PV}/${MY_P}-src.tar.gz"
+
+# Sabnzbd is GPL-2 but bundles software with the following licenses.
+LICENSE="GPL-2 BSD LGPL-2 MIT BSD-1"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+7za +rar unzip test"
+RESTRICT="!test? ( test )"
+
+# Is this still correct? The article has disappeared from the wiki!
+# Sabnzbd is installed to /usr/share/ as upstream makes it clear they should not
+# be in python's sitedir.  See: https://sabnzbd.org/wiki/advanced/unix-packaging
+
+COMMON_DEPS="
+	acct-user/sabnzbd
+	acct-group/sabnzbd
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cheetah3[${PYTHON_USEDEP}]
+		dev-python/cherrypy[${PYTHON_USEDEP}]
+		dev-python/configobj[${PYTHON_USEDEP}]
+		dev-python/cryptography[${PYTHON_USEDEP}]
+		>=dev-python/feedparser-6.0.10[${PYTHON_USEDEP}]
+		>=dev-python/guessit-3.1.0[${PYTHON_USEDEP}]
+		dev-python/notify2[${PYTHON_USEDEP}]
+		dev-python/portend[${PYTHON_USEDEP}]
+		dev-python/puremagic[${PYTHON_USEDEP}]
+		>=dev-python/sabyenc-5.4.2[${PYTHON_USEDEP}]
+	')
+"
+
+DEPEND="${COMMON_DEPS}"
```

Comment:
`COMMON_DEPS` looks useless in here, since no differences are defined in `DEPEND`. You can just use `DEPEND`.